File: Secao06 - Estruturas de Decisão/Exercicios em Python/Exercicio07.py
# -*- coding: utf-8 -*-
print("Numeros ao Quadrado.\n")
num1 = int(input("Informe o 1º numero: "))
num2 = int(input("Informe o 2º numero: "))
num3 = int(input("Informe o 3º numero: "))
num4 = int(input("Informe o 4º numero: "))
print(" ")

q1 = num1 * num1
q2 = num2 * num2
q3 = num3 * num3
q4 = num4 * num4

if q3 > 1000:
    print("Quadrado apenas do Terceiro valor = {0}.".format(q3))
else:
    print("Quadrado de todos os numeros:")
    print("{0} = {1}".format(num1, q1))
    print("{0} = {1}".format(num2, q2))
    print("{0} = {1}".format(num3, q3))
    print("{0} = {1}".format(num4, q4))

File: Secao06 - Estruturas de Decisão/Exercicios em Java/Exercicio07.java
package br.com.udemy.secao06;

import java.util.Scanner;

public class Exercicio07 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int num1, num2, num3, num4, q1, q2, q3, q4;
		
		System.out.print("Informe 1º numero inteiro: ");
		num1 = teclado.nextInt();
		System.out.print("Informe 2º numero inteiro: ");
		num2 = teclado.nextInt();
		System.out.print("Informe 3º numero inteiro: ");
		num3 = teclado.nextInt();
		System.out.print("Informe 4º numero inteiro: ");
		num4 = teclado.nextInt();
		
		q1 = num1 * num1;
		q2 = num2 * num2;
		q3 = num3 * num3;
		q4 = num4 * num4;
		
		if(q3 >= 1000) {
			System.out.println("O quadrado do Terceiro Numero é: " + q3);
		}else{
			System.out.println(+ num1 + " - " + q1);
			System.out.println(+ num2 + " - " + q2);
			System.out.println(+ num3 + " - " + q3);
			System.out.println(+ num4 + " - " + q4);
			
		}
		
		teclado.close();

	}

}
